.github/workflows/continuous-integration.yml

```diff
@@ -34,19 +34,19 @@ jobs:
               run: npm install --legacy-peer-deps
 
             - name: Run the lints
-              run: npm run lint
+              run: yarn lint
```

Comment:
Why did you choose the Yarn instead of NPM? since npm v7 is much better than yarn.

Reply:
I agree that npmv7 is good, but I prefer to use `yarn` because still it is much faster and simpler in case of installing and command simplicity(than current stable npm), also it has some valuable features like `workspace` which can be used later and is popularly used with other tools. Also current stable  `nodejs v14.17.1LTS` does not include `npm v7` by default, So we can replace it whenever the new version of npm has been stabled and arrived with nodejs itself. Npm v7, has support for `yarn.lock` too and Since we can remove it or replace it any time( or roll back to npm), I think there is no need to worry about that.
What is your opinion?

Reply:
As I researched before, npm 7 supports the workspace now and is much faster than before. I think there is no need to replace that whereas we know that we want to replace it with the NPM in the future again.